examples/experimental/test/end_to_end/candid_rpc/async_await/package.json

```diff
@@ -8,6 +8,6 @@
     },
     "dependencies": {
         "azle": "0.31.0",
-        "azle-experimental-deps": "github:demergent-labs/azle-experimental-deps"
+        "azle-experimental-deps": "github:demergent-labs/azle-experimental-deps#34cf2e34958e03cae7bf63f77f86ae682eef8a7c"
```

Comment:
Check that installing with the https url doesn't change this, if it does let's change

Reply:
it doesn't